Packs/CommonPlaybooks/ReleaseNotes/2_4_26.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### User Investigation - Generic
+
+- Fixed an issue where Cortex Core was not utilized in the user investigation in XSIAM.
+- Fixed an issue with the PAN-OS firewall search.
```

Comment:
What was the issue?  I think this needs more information.

Reply:
Fixed